MantidQt/CustomInterfaces/src/MultiDatasetFit/MultiDatasetFit.cpp

```diff
@@ -713,5 +714,11 @@ void MultiDatasetFit::updateGuessFunction(const QString &, const QString &) {
   m_plotController->updateGuessFunction(*m_functionBrowser->getFunction());
 }
 
+/// Log a warning
+/// @param msg :: A warning message to log.
+void MultiDatasetFit::logWarning(const std::string &msg) {
+  g_log.warning() << msg << std::endl;
```

Comment:
Not that it makes any real difference, bit forma consistency `\n` would be preferred. Also, the message can be passed as a parameter rather than using the stream operators.